core/commands/inline_diff.py

```diff
@@ -289,6 +289,16 @@ def highlight_regions(self, replaced_lines):
 
         self.view.add_regions("git-better-added-lines", add_regions, scope="git_savvy.change.addition")
         self.view.add_regions("git-better-removed-lines", remove_regions, scope="git_savvy.change.removal")
+        self.scroll_to_the_first_change(l)
```

Comment:
Instead of taking the action here in `highlight_regions`, move the call to `scroll_to_first_change` up [here](https://github.com/NicoSantangelo/GitSavvy/blob/scroll-to-first-diff/core/commands/inline_diff.py#L150) right after the call to `highlight_regions`.  You should have access to the relevant line number as part of the `replaced_lines` list.